finagle-redis/src/main/scala/com/twitter/finagle/redis/HyperLogLogCommands.scala

```diff
@@ -0,0 +1,47 @@
+package com.twitter.finagle.redis
+
+import _root_.java.lang.{Long => JLong,Boolean => JBoolean}
```

Comment:
Looks unnecessary to me as well, but it's in all the other `*Commands.scala` files, too.

There's a fair amount of noise and historical baggage in finagle-redis, and since it's more or less in maintenance mode until we move it to the Finagle org it's been a relatively low priority. So we have to decide between 1) doing the right thing here and being inconsistent, 2) cleaning up all instances of this at once, or 3) leaving it as is. I'm fine with either 1) or 3), but think 2) should happen in a new PR if at all.